### PR TITLE
Update /support overview

### LINKED ISSUE #4703: UA for infrastructure - Update support overview

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Updated /support for UA for Infrastructure

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support](http://0.0.0.0:8001/support)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1liY2ulrI3JiICSJquTn9cH0Hu-R6H3ufmEmYhive-QI/edit#)

## Issue / Card

Fixes #4703

### CODE REVIEW

File: templates/support/index.html [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+      <h1>The Ubuntu Advantage</h1>
+      <h4>Open Source support for the enterprise</h4>
+      <p>Ubuntu Advantage for your infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry. With OpenStack and Kubernetes support included, Ubuntu Advantage for Infrastructure (UA-I) provides everything you need to future-proof your data centre.</p>
+      <p>Including Canonical Livepatch, Landscape and Extended Security Maintenance (ESM) to address security and compliance concerns, Ubuntu Advantage for Infrastructure provides a comprehensive single pane of glass for systems management, audit reporting and live kernel patching to prevent unplanned maintenance and downtime.</p>
       <p>
         <a href="https://buy.ubuntu.com" class="p-button--positive"><span class="p-link--external">Visit the store</span></a>
         <a href="{{ ASSET_SERVER_URL }}63deda69-canonical-ua-datasheet-2018-09-14.pdf" class="p-button--neutral">Get the Ubuntu Advantage datasheet</a>
```
Comment (anthonydillon) [inline templates/support/index.html]: This should be "Read the Ubuntu Advantage datasheet"

File: templates/shared/_case-study-allan-gray.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<section class="p-strip is-deep is-bordered">
+        <div class="row u-equal-height">
+          <div class="col-8">
+            <h2>Discover why Allan Gray turned to Canonical to boost efficiency and availability</h2>
+            <p>Allan Gray needed to ensure maximum availability for mission-critical systems, which is why it turned to Canonical for enterprise support for Kubernetes.</p>
+            <a class="p-button--neutral" href="/engage/k8s-allan-gray">Read the case study</a>
```
Comment (anthonydillon) [inline templates/shared/_case-study-allan-gray.html]: The other case study include should also link to the engage page not "https://pages.ubuntu.com/Cloud_CS_ITstrategen.html"

File: templates/support/index.html [hunk 3]
```diff
@@ -95,46 +95,25 @@ <h3 class="p-matrix__title p-heading--four">Ubuntu Advantage for public cloud</h
...
-      <p>Reduce your team&rsquo;s efforts on day-to-day management with Landscape, the most cost-effective tool to support and monitor large and growing networks of&nbsp;desktops, servers and clouds.</p>
+      <h2>Infrastructure solutions</h2>
+      <h3>Systems management and security at scale</h3>
+      <p>Automate day-to-day management with Landscape, the most cost-effective tool to support and monitor large and rapidly expanding networks of clouds, servers and desktops. Use Landscape for audit reporting, security compliance and patch management at scale. Landscape is available as a service from the cloud or on-premises for compliance with stricter security requirements.</p>
       <p>Take control of your infrastructure with:</p>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">Automated updates and managed physical, virtual and cloud-based systems from a single interface</li>
+        <li class="p-list__item is-ticked">Automate updates and manage physical, virtual and cloud-based systems from a single interface</li>
         <li class="p-list__item is-ticked">Monitored and managed machines at scale</li>
```
Comment (anthonydillon) [inline templates/support/index.html]: This one should be "Monitor and manage machines at scale"

File: templates/support/index.html [hunk 4]
```diff
@@ -95,46 +95,25 @@ <h3 class="p-matrix__title p-heading--four">Ubuntu Advantage for public cloud</h
...
+        <li class="p-list__item is-ticked">Maintain security and compliance</li>
         <li class="p-list__item is-ticked">Controlled inventory</li>
         <li class="p-list__item is-ticked">Integration with current systems</li>
```
Comment (anthonydillon) [inline templates/support/index.html]: This one should be "Integrate with current systems"

**Comment by anasereijo** [comment]
please amend the following:

support and management ubuntu

support and management ubuntu 1

support and management ubuntu 2

**Comment by anasereijo** [comment]
@pmahnke thanks for making the changes.
I just noticed that at&t logo isn't displaying properly. Can you fix it?

support and management ubuntu 3

**Comment by anasereijo** [comment]
@pmahnke the logo looks the same

**Comment by pmahnke** [comment]
@anasereijo 	trying a png

**Comment by anasereijo** [comment]
LGTM 👍

**Pull request merged**